_rules/SC4-1-2-role-attribute-has-valid-value.md

```diff
@@ -0,0 +1,114 @@
+---
+name: Role attribute has valid value
+test_type: atomic
+
+description: |
+   This rule checks that elements with a role attribute have valid values
+   
+success_criterion:
+- 4.1.2 # Name, Role, Value
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+Any [non-empty](#non-empty) `role` attribute that is placed on an HTML or SVG element that is [exposed to assistive technologies](#exposed-to-assistive-technologies).
+
+### Expectation
+
+Each test target has at least one valid value as specified in the [WAI-ARIA 1.1, roles](https://www.w3.org/TR/wai-aria-1.1/#roles_categorization).
+
+## Assumptions
+
+*There are currently no assumptions*
+
+## Accessibility Support
+
+Older browsers do not support fallback roles. If multiple roles are used in the role attribute, the attribute is ignored.
```

Comment:
"Older browsers do not support fallback roles" - can we get a link or a definition for "fallback roles"?

"If multiple roles are used in the role attribute, the attribute is ignored." --> "If multiple **values** are used in the role attribute, the attribute is ignored." - suggesting this  to keep it consistent with the wording in the Expectation

Reply:
Updated as per suggestion.

I had no luck finding a link for `fallback roles`. The WAI Aria spec - https://www.w3.org/TR/wai-aria-1.1/ talks about fallback in general. 

@annethyme - if you could find a link do share.

But I believe the word fallback is self explanatory and a definition might be an overkill at the moment.